### PR TITLE
replace react-use-gesture to @use-gesture/react

### PR DESCRIPTION
**Pull request by wey-gu**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] Other... Please describe:
```

## What is the current behavior?

react-use-gesture is not maintained and @use-gesture/react is the successor

Issue Number: N/A

## What is the new behavior?

Warning of react-use-gesture during `npm i`

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ghsteff — APPROVED** [review]
Tested dragging on desktop and mobile 👍

**Review by ghsteff — CHANGES_REQUESTED** [review]
Oops was testing on the wrong branch 😅 dragging nodes doesn't seem to be working

**Comment by ghsteff** [comment]
Here's a video of the `Nodes/Draggable` story. Is this working correctly on your machine @wey-gu ?

https://github.com/user-attachments/assets/2dc0a889-d104-412e-8931-9fe74accdfe0

File: src/utils/useDrag.ts [hunk 1]
```diff
@@ -63,10 +63,19 @@ export const useDrag = ({
...
+          clientY = event.clientY;
+        } else if (event instanceof TouchEvent) {
+          clientX = event.touches[0].clientX;
+          clientY = event.touches[0].clientY;
+        } else {
+          return;
+        }
+
+        const nx = ((clientX - (clientRect?.left ?? 0)) / size.width) * 2 - 1;
+        const ny = -((clientY - (clientRect?.top ?? 0)) / size.height) * 2 + 1;
```
Comment (ghsteff) [inline src/utils/useDrag.ts:78]: Hmmm playing around with this and the types for `event` seem wrong. It's hitting the `else { return }` block every time

This should work with no ts complaints tho

```
      onDrag: ({ xy }) => {
        // Compute normalized mouse coordinates (screen space)
        const nx = ((xy[0] - (clientRect?.left ?? 0)) / size.width) * 2 - 1;
        const ny = -((xy[1] - (clientRect?.top ?? 0)) / size.height) * 2 + 1;
```

Comment (wey-gu) [reply src/utils/useDrag.ts:78]: Will do tomorrow, thanks!🫡

**Review by ghsteff — APPROVED** [review]
🚀

**Pull request merged**